extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -1222,6 +1272,38 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
         return null
       }
 
+    /**
+     * Whether to generate ids for the geojson features. When enabled, the `feature.id` property will be auto
+     * assigned based on its index in the `features` array, over-writing any previous values.
+     */
+    val defaultGenerateId: Boolean?
```

Comment:
Is this property needed? What's the difference between this and generateId?

Reply:
This is the static getter for the default value for the generateId property. By default I think this field is false. The generatedId property is useful for feature state API.